src/test/kotlin/uk/gov/dluhc/printapi/config/IntegrationTest.kt

```diff
@@ -38,7 +52,32 @@ internal abstract class IntegrationTest {
         TestLogAppender.reset()
     }
 
+    @BeforeEach
+    fun clearDatabase() {
+        clearTable(dynamoDbConfiguration.printDetailsTableName)
+    }
+
     companion object {
         val localStackContainer = LocalStackContainerConfiguration.getInstance()
     }
+
+    protected fun clearTable(tableName: String, partitionKey: String = "id", sortKey: String? = null) {
```

Comment:
Originally I thought this might add a bit too much to the processing, but on reflection the Dynamo tables are going to be largely empty so that shouldn't be a concern for the Integration Tests.  At some point should port this over to the notifications api 👍